sig-cli/CONTRIBUTING.md

```diff
@@ -97,35 +90,24 @@ work.
   - Add `kubectl hello-kubernetes -f file`: Print "Hello \<kind of resource\> \<name of resource\>"
   - Add `kubectl hello-kubernetes type/name`: Print "Hello \<kind of resource\> \<name of resource\> \<creation time\>"
 
-**Note:** Consider publishing your command to a fork so a maintainer can look at it.
+Push your branch to your github fork (`git push -f origin
+{branchName}`) so a maintainer can look at it.
 
 ## Your first contribution
 
 ### Adopt an issue
 
-Pick up an [issue] from the backlog by commenting on the issue that you would like to work on it.
-Be sure to mention the author of the issue as well as the SIG cli members `@seans3` and `@mengqiy`.
-
-Using the following comment will make it easier for us to search for issues folks want to have
-assigned to them:
-
-`cc @seans3 @mengqiy I would like to take this`
+Browse the [issues] for something you'd like to work on.
+The label "[good first issue]" has been applied to
+issues that might be more appropriate for newcomers.
 
-**Note:** Don't do this unless you will start work on the issue within a few days of being assigned.
+An alternative and interesting presentation
```

Comment:
No sure how helpful this will be to new contributors.  Is that work regularly groomed, up to date and well thought out enough for new contributors to pick up arbitrary issues from it?

Reply:
There's a "good first issue" query linked to earlier in the text that serves that purpose.  The text implies those issues are is curated, and that the wiki is merely alternative and interesting.

I like the idea of a page that provides an issue overview / story / broad taxonomy that doesn't require one to both know all the labels and how to use them https://github.com/kubernetes/kubectl/labels
Brian mentions it frequently, but he and Janet seemed to have stopped maintaining it last year after ~100 edits. Nothing in community points to it, so i thought we could give it a rank boost given its clearly sig-cli.

Perhaps we could file an issue to replace this wiki with human readable sentences or headings that link to pre-baked issue queries with particular labels.

Then its maintenance would be no worse that whatever effort is currently being put into applying labels.

regardless, removed link